models/addCommentToDb.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection.js");
+
+function addCommentToDb({ body, votes, author, article_id, created_at }) {
+  return db
+    .query(
+      `INSERT INTO comments (body, article_id, author, votes, created_at)
+        VALUES ($1, $2, $3, $4, $5)
+        RETURNING *`,
+      [body, article_id, author, votes, created_at]
+    )
```

Comment:
Good query string but votes and created_at has default value, so not needed here